js/binance.js

```diff
@@ -4547,6 +4549,131 @@ module.exports = class binance extends Exchange {
         };
     }
 
+    async fetchDepositWithdrawFees (codes = undefined, params = {}) {
+        /**
+         * @method
+         * @name binance#fetchDepositWithdrawFees
+         * @description fetch deposit and withdraw fees
+         * @param {[string]|undefined} codes not used by binance fetchDepositWithdrawFees ()
+         * @param {object} params extra parameters specific to the binance api endpoint
+         * @returns {[object]} a list of [fee structures]{@link https://docs.ccxt.com/en/latest/manual.html#fee-structure}
+         */
+        await this.loadMarkets ();
+        const response = await this.sapiGetCapitalConfigGetall (params);
```

Comment:
We can remove `await ...sapi` call there, as `loadMarkets` already does that through `fetchCurrencies`. So, no need to add extra await here, you can just access `this.currencies` and iterate through their `info` values

Reply:
we should remove the `loadMarkets` then, [check these comments](https://github.com/ccxt/ccxt/pull/15581#discussion_r1014869094)

Reply:
I've seen those comments and right, I've changed my mind, we should not use cached data from .currencies, as the transaction fees might have changed and user should get up-to-date data. `loadMarkets` also doesnt hurt much as it happens only once. so, I'd retract my above comment, ignore that and continue as is👍

Reply:
Instead, maybe we can do `this.fetchCurrencies()` instead of `await this.loadMarkets (); && await this.sapiGetCapitalConfigGetall (params);` and then use `this.currencies` there? 
it has advantage, because it will re-load currencies too (in case there was change in currency list too) as long as we call the same endpoint as currencies, why not catch two rabbits with one shot?